### PR TITLE
Added TF import tests

### PR DESCRIPTION
**Pull request by alexanderst**
Added some TF import tests

### CODE REVIEW

File: import-tests/tests/test_ops.py [hunk 1]
```diff
@@ -1256,11 +1257,17 @@ def test_mathtransform():
         # {"opName": "fake_quant_with_min_max_vars_per_channel", "outName": "fake_quant/min_max_args_per_channel/rank2_8bit_narrow", "varShapes":[[3, 5], [5], [5]], "varTypes":["float32", "float32", "float32"], "varInit":["uniform", "uniform_m1_0", "uniform"], "num_bits":8, "narrow_range":True},
         # {"opName": "fake_quant_with_min_max_vars_per_channel", "outName": "fake_quant/min_max_args_per_channel/rank4_6bit", "varShapes":[[3, 2, 2, 5], [5], [5]], "varTypes":["float32", "float32", "float32"], "varInit":["uniform", "uniform_m1_0", "uniform"], "num_bits":6, "narrow_range":False},
 
+        #{"opName": "adjust_saturation", "outName": "adjust_saturation/rank3_float32", "varShapes": [[8, 8, 3]], "varTypes": ["float32"], "varInit": ["uniform"], "factor": 2.0},
```
Comment (AlexDBlack) [inline import-tests/tests/test_ops.py]: One more test here would be good, rank 4, saturation = 0.5

File: import-tests/tests/test_ops.py [hunk 2]
```diff
@@ -416,6 +415,8 @@ def test_mathtransform():
         # {"opName":"max_pooling1d", "outName":"max_pooling1d/channels_last_b2_k2_s1_VALID", "varShapes":[[2, 5, 2]], "varTypes":["float32"], "pooling_size":2, "stride":1, "padding":"VALID", "data_format":"channels_last"},
         # {"opName":"max_pooling1d", "outName":"max_pooling1d/channels_last_b1_k2_s2_SAME", "varShapes":[[1, 5, 2]], "varTypes":["float32"], "pooling_size":2, "stride":2, "padding":"SAME", "data_format":"channels_last"},
 
+        #{"opName":"max_pool_with_argmax", "outName":"max_pool_with_argmax/rank1", "varShapes":[[1, 2, 2, 5]], "varTypes":["float32"], "varInit":["uniform"], "ksizes":[1,2],"strides":[1,1], "padding":"SAME", "data_format":"NHWC", "output_dtype":tf.int32},
```
Comment (AlexDBlack) [inline import-tests/tests/test_ops.py]: Need more tests for this.
For example: double, NCHW, int64 datatype, valid padding mode, etc.

File: import-tests/tests/test_ops.py [hunk 3]
```diff
@@ -2045,6 +2052,42 @@ def test_mathtransform():
...
+
+         #{"opName": "matrix_band_part", "outName": "matrix_band_part/float32", "varShapes": [[3,4]],  "varTypes": ["float32"], "varInit": ["uniform"],"num_lower":1, "num_upper":-1},
+         #{"opName": "matrix_band_part", "outName": "matrix_band_part/float64", "varShapes": [[3, 4]],  "varTypes": ["float64"], "varInit": ["uniform"], "num_lower": 1, "num_upper": -1},
+         #{"opName": "matrix_band_part", "outName": "matrix_band_part/float32_reverse_limits", "varShapes": [[3, 4]],   "varTypes": ["float32"], "varInit": ["uniform"], "num_lower": 0, "num_upper": 1},
+
+         #{"opName": "polygamma", "outName": "polygamma/float32", "varShapes": [[3,4],[3,4]],  "varTypes": ["float32","float32"], "varInit": ["uniform","uniform"]},
+         #{"opName": "polygamma", "outName": "polygamma/float64", "varShapes": [[3, 4], [3, 4]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+         #{"opName": "polygamma", "outName": "polygamma/emptyArrayTest/float32", "varShapes": [[0, 0], [3, 4]],  "varTypes": ["float32", "float32"], "varInit": ["empty", "uniform"]},
```
Comment (AlexDBlack) [inline import-tests/tests/test_ops.py:2079]: Fix initialization for first input - it should be integer values (but float type)
https://github.com/eclipse/deeplearning4j/issues/8452#issuecomment-558880111
https://www.tensorflow.org/api_docs/python/tf/math/polygamma

File: import-tests/tests/test_ops.py [hunk 4]
```diff
@@ -2045,6 +2052,42 @@ def test_mathtransform():
...
+
+         #{"opName": "random_crop", "outName": "random_crop/float32", "varShapes": [[4,2],[2]],"varTypes": ["float32", "int32"], "varInit": ["stdnormal", "uniform_int2"]},
+         #{"opName": "random_crop", "outName": "random_crop/float64", "varShapes": [[4, 2], [2]],  "varTypes": ["float64", "int32"], "varInit": ["stdnormal", "uniform_int2"]},
```
Comment (AlexDBlack) [inline import-tests/tests/test_ops.py]: Make first inputs bigger... [4,2] for images is too small. Say [8,8]
Also both 3d (say [[8,8,3],[3] with fixed values (4,4,3)) and 4d ([[1,8,8,3],[4]])

File: import-tests/tests/test_ops.py [hunk 5]
```diff
@@ -2045,6 +2052,42 @@ def test_mathtransform():
...
+
+         #{"opName": "roll", "outName": "roll/rank1_float32", "varShapes": [[4]],"varTypes": ["float32"], "varInit": ["uniform"], "shift":2, "axis":0},
+         #{"opName": "roll", "outName": "roll/rank1_float64", "varShapes": [[4]], "varTypes": ["float64"], "varInit": ["uniform"], "shift": 2, "axis": 0},
+         #{"opName": "roll", "outName": "roll/rank2_float32", "varShapes": [[2,4]], "varTypes": ["float32"],  "varInit": ["uniform"], "shift": 2, "axis": 0},
+         #{"opName": "roll", "outName": "roll/rank2_float32_zeroshift", "varShapes": [[2, 4]], "varTypes": ["float32"],  "varInit": ["uniform"], "shift": 0, "axis": 1},
```
Comment (AlexDBlack) [inline import-tests/tests/test_ops.py:2088]: Add Rank 2 case with shape [4,5] and non-zero cases for both axis.
Rank 3 and 4 cases, with different axis.
For the new rank3/4 cases, make some of them different datatypes - integer, boolean, etc

**Review by AlexDBlack — CHANGES_REQUESTED** [review]
Improved coverage and one initialization fix needed, but otherwise good.

File: import-tests/tests/test_ops.py (hunk 4 above)
Comment (alexanderst) [reply import-tests/tests/test_ops.py]: Only 3D arrays worked.

File: import-tests/tests/test_ops.py (hunk 3 above)
Comment (alexanderst) [reply import-tests/tests/test_ops.py:2079]: Done.

File: import-tests/tests/test_ops.py (hunk 5 above)
Comment (alexanderst) [reply import-tests/tests/test_ops.py:2088]: Done

**Review by AlexDBlack — APPROVED** [review]
:+1:

**Pull request merged**